lib/Dialect/TritonGPU/IR/Dialect.cpp

```diff
@@ -178,6 +187,10 @@ SmallVector<unsigned> getSizePerThread(Attribute layout) {
     } else {
       llvm_unreachable("Unexpected mma version");
     }
+#ifdef USE_ROCM
+  } else if (auto mfmaLayout = layout.dyn_cast<MfmaEncodingAttr>()) {
+    return {4, 4};
```

Comment:
Can you please explain why is here shape {4, 4}? I believe the shape should be {16, 1} based on a layout of C/D matrices mfma layout. Every thread will take indices from the same column, but across 16 different rows.

Reply:
Your suggestion looks valid, I think this code was not used before (at least, separate axis info), it did not break anything.